src/components/Home.jsx

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from 'react';
+import ArticleCards from './ArticleCards';
+import { getArticles } from '../api';
+
+export default function Home() {
+    const [articles, setArticles] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+    const [error, setError] = useState(null);
+    
+    useEffect(() => {
+        getArticles()
+        .then((articlesFromApi) => {
+            const sortedArticles = articlesFromApi.sort((a, b) => a.article_id - b.article_id);
+            setArticles(sortedArticles);
+            setIsLoading(false);
```

Comment:
How necessary is this `sort` method, by default the articles come back sorted by date from the API